cmd/run.go

```diff
@@ -99,6 +100,11 @@ dapr run --app-id myapp --app-port 3000 --app-protocol grpc -- go run main.go
 			}
 		}
 
+		internalGrpcPort, err := getFreePort()
```

Comment:
Can you also change the implementation to allow passing in internal grpc port from cmd flag? Wrt issue #1026